_data/navigation.yml

```diff
@@ -12,6 +12,9 @@
 primary:
   - text: Documentation
     href: /docs/
+  - text: Button
+    href: /button/
+    type: Button
```

Comment:
If you add `type: button` or `type: Button` the class `usa-button` will be added to the link